queries/builtins.ts

```diff
@@ -0,0 +1,217 @@
+/*! *****************************************************************************
```

Comment:
Do we plan on editing this file in the future, or is it a copy of a file from elsewhere?  If the latter, can you add

```
queries/builtins.ts  -diff -whitespace linguist-vendored
```

to .gitattributes?

Reply:
It's not a direct copy of one file. I cherry picked the most impactful parts from various TypeScript standard library definitions. I guess it should remain as-is then?